webapp/channels/src/components/suggestion/emoticon_provider.test.tsx

```diff
@@ -34,8 +39,8 @@ describe('components/EmoticonProvider', () => {
     });
 
     it('should suggest emojis when partial name >= MIN_EMOTICON_LENGTH', () => {
-        getEmojiMap.mockReturnValue(emojiMap);
-        getRecentEmojisNames.mockReturnValue([]);
+        (getEmojiMap as unknown as jest.Mock).mockReturnValue(emojiMap);
+        (getRecentEmojisNames as unknown as jest.Mock).mockReturnValue([]);
```

Comment:
Instead of this, use the `mocked` pattern: https://jestjs.io/docs/mock-function-api/#jestmockedsource-options

So... something like...
```
jest.mock('selectors/emojis', () => ({
    ...
}));
const mockedGetEmojiMap = jest.mocked(getEmojiMap);
...
it(..., () => {
    mockedGetEmojiMap.mockReturnValue(emojiMap);
    ...
```

Reply:
Hello @larkox ,
I updated the file to have the suggested changes. Please re-review.